app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -405,12 +419,22 @@ def handle_unsupported_secondary_id(enrollment, response)
     )
   end
 
+  # MW: This could be inlined but it feels a little cleaner this way:
+  def fraud_pending?(enrollment)
+    enrollment.profile&.fraud_pending_reason.present?
+  end
```

Comment:
These comments won't stay in the PR; they were reminders to me to start them here.

We have `fraud_pending_reason` and `fraud_review_pending_at`, so `fraud_pending?` is potentially ambiguous. There is a lot of bikeshedding that can be done on this name and I'm open to it.

This method is really here for readability than because it makes sense as a standalone method. (Arguably it would be more proper on Profile, but a method there that just checks whether another attribute is present feels silly.) It's more expressive to call `fraud_pending?` in a few places than to sprinkle `enrollment.profile&.fraud_pending_reason.present?` around.

Reply:
how about something that indicates the result of the fraud review is pending. Like `fraud_adjudication_pending`, `fraud_result_pending`, `fraud_finding_pending`, etc?

Reply:
I like these. Updated!